containers/base.Dockerfile

```diff
@@ -68,6 +70,7 @@ FROM builder-base as poetry-deps-install-amd64
 
 WORKDIR /pixelator
 COPY poetry.lock pyproject.toml ./
+COPY .git .git
```

Comment:
Didn't we talk about setting the via an environment variable instead? I haven't investigated it in practice, but I guess one thing we could do is use something like the below from the poetry dynamic versioning docs (https://pypi.org/project/poetry-dynamic-versioning/):

env `POETRY_DYNAMIC_VERSIONING_OVERRIDE="pixelator={{VERSION BASED ON GITHUB CI VARS}}"`.

On the other hand since the git folder is not included in the final container image anyway maybe this approach is easier?

Reply:
I looked into using POETRY_DYNAMIC_VERSIONING_OVERRIDE like we do in eg. conda recipes, but I found this to be much more straigthforward. Especially when using this as a base image for pixelator-internal where we do want the detailed versioning.

I will add the option to still override the version with POETRY_DYNAMIC_VERSIONING_OVERRIDE using a build arg.